### PR TITLE
NOJIRA Change transpilation target from CJS to ESM

### PR DESCRIPTION
**Pull request by xalechez**
Changing transpilation target from CommonJS to ES Modules to fix issues with tree shaking and increasing bundle size

Remember to include the following changes:

- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ module.exports = {
         targets: {
           node: 'current',
         },
+        modules: false,
```
Comment (runmoore) [inline babel.config.js:34]: [Context](https://babeljs.io/docs/babel-preset-env#modules)

> Setting this to false will preserve ES modules. Use this only if you intend to ship native ES Modules to browsers. If you are using a bundler with Babel, the default modules: "auto" is always preferred.

**Review by olliecurtis — APPROVED** [review]

File: babel.config.js [hunk 2]
```diff
@@ -77,6 +73,21 @@ module.exports = {
       ],
     },
     test: {
+      presets: [
```
Comment (xalechez) [inline babel.config.js:76]: Preserving old environment for jest so it doesn't break

File: packages/bpk-component-radio/src/BpkRadio.module.css [hunk 3]
```diff
@@ -15,4 +15,4 @@
...
...ckground:#fff}.bpk-radio--disabled{color:rgba(0,0,0,0.2);cursor:not-allowed}.bpk-radio--disabled--white{color:rgba(255,255,255,0.5)}.bpk-radio--invalid input[type='radio']{border:3px solid #e70866;border-radius:1.5rem}.bpk-radio__input{position:absolute;top:0;left:0;width:1.25rem;height:1.25rem;margin:0;padding:0;cursor:pointer;vertical-align:text-bottom;appearance:none;border:3px solid #545860;border-radius:1.5rem}html[dir='rtl'] .bpk-radio__input{right:0;left:auto}@media screen\0 {.bpk-radio__input{background:none !important}}.bpk-radio__input:checked{border-color:#0062e3;border-color:var(--bpk-radio-checked-color, #0062e3)}.bpk-radio__input:checked:disabled{border-color:rgba(0,0,0,0.2)}.bpk-radio__input:disabled{border-color:rgba(0,0,0,0.2)}.bpk-radio__input:disabled{cursor:inherit}.bpk-radio__input:checked+.bpk-radio__circle{display:block}.bpk-radio__input:checked:disabled+.bpk-radio__circle{display:block;background:rgba(0,0,0,0.2)}.bpk-radio__circle{position:absolute;top:0.375rem;left:0.375rem;display:none;width:.5rem;height:.5rem;border-radius:50%;background:#0062e3;background:var(--bpk-radio-checked-color, #0062e3)}html[dir='rtl'] .bpk-radio__circle{right:0.375rem;left:auto}
```
Comment (runmoore) [inline packages/bpk-component-radio/src/BpkRadio.module.css:18]: What has changed in this 1 line? is it a separate change?

One for another time, but It looks like bundled css and surprised to see these checked in. I'd be interested to know if we can leave the generation of these to build time.

Comment (olliecurtis) [reply packages/bpk-component-radio/src/BpkRadio.module.css:18]: We can totally leave them there and exclude them if we want currently our process just has them in and we copy them over as part of our process.

This acts more of a sanity check of do we expect these to change

**Pull request merged**